homeassistant/components/cast/manifest.json

```
@@ -4,7 +4,7 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/cast",
   "requirements": ["pychromecast==7.2.1"],
-  "after_dependencies": ["cloud","tts","zeroconf"],
+  "after_dependencies": ["cloud", "http", "media_source", "tts", "zeroconf"],
```

This should just be `media_source`. 
```suggestion
  "after_dependencies": ["cloud", "media_source", "tts", "zeroconf"],
```

---

Ah hah, that's why `http` was added. hassfest failed without it!